### DISCUSSION TITLE
How To Contribute to the Price-Transparency-Guide?

### DISCUSSION THREAD

**Discussion by jkHCBB** [CMSgov/price-transparency-guide#734] [Q&A] [answered]
@shaselton-usds I have a question,

If as a user I wanted to contribute new code or features that I built/wrote in here or in the price-transparency-guide-validator repo:
1. Is this allowed?
2. Are there any guidelines/steps in place that need to be taken in order to make this happen?

I would love to be able to help out here and in the validator repo where I can.

Thank you

**Comment by shaselton-usds** [comment, marked as answer]
Hi @jkHCBB 

Thanks for reaching out. You can always make suggestions on how you'd structure the schema, additional tooling that you think might help, etc.

You're also more than welcome to fork or clone this repo and build whatever you'd like in your space too.

Development on the validator is a bit more flexible as it is not constrained by regulation or rulemaking. Any type of feature request or development can start out as a discussion or you can clone the repo, make changes and propose a PR into the main validator repo.

Thanks again for your interest!

**Reply by jkHCBB** [reply]
Thank you for the reply! I appreciate the information and will try my best to help out.
Cheers
